main.go

```diff
@@ -100,6 +101,7 @@ func main() {
 	flag.IntVar(&maxAPIRetries, "max-api-retries", 12, "The number of maximum retries for failed/rate limited AWS API calls")
 	flag.IntVar(&drainTimeout, "drain-timeout", 900, "when the drain command should timeout")
 	flag.BoolVar(&ignoreDrainFailures, "ignore-drain-failures", false, "proceed with instance termination despite drain failures.")
+	flag.IntVar(&maxReplacementNodes, "max-replacement-nodes", 5, "The max number of replacement nodes allowed in a cluster. Avoids cluster-ballooning")
```

Comment:
Is it a good idea to set the default to 5?
Default should be 0 (no limit) otherwise people would have a breaking change

Reply:
Oops. That was a default I used during testing. Yes. The idea is to set the default to -1

Reply:
Should we use 0 instead of -1 to indicate no limit? this would align better with other options such as drain-timeout, etc. and also 0 is not a valid option

Reply:
Sure. I can set it to 0, to keep it uniform.